assets/js/vue/components/search/ResultSet.vue

```diff
@@ -89,15 +90,15 @@
                         queryParam: 'status'
                     },
                     fundingCycle: {
-                        label: 'Funding Cycles',
+                        label: (PELAGOS_TEMPLATE_PROPS.BaseTemplateName === 'GRP' ? "Grant Awards" : "Funding Cycles"),
```

Comment:
It's better to move this to methods (line 134), that way you can only check if `PELAGOS_TEMPLATE_PROPS` is defined.
Currently this only works because `BaseTemplateName` is defined in the GRIIDC template, where is has no function at all.